rpc/src/v1/impls/light/eth.rs

```diff
@@ -493,6 +493,12 @@ impl<T: LightChainClient + 'static> Eth for EthClient<T> {
 		}))
 	}
 
+
+	fn proof(&self, address: RpcH160, values:Vec<RpcH256>, num: Trailing<BlockNumber>) -> BoxFuture<EthAccount> {
+		Box::new(future::err(errors::unimplemented(None)))
```

Comment:
Intentional?

Reply:
At the moment getting a proof within the light-client would require a full state, but I agree we would need to implement this in the light_fetcher to retrieve the proof from a peer with a full state. This still needs to be done.